web/webkit/src/main/scala/net/liftweb/http/SHtml.scala

```diff
@@ -169,6 +169,21 @@ trait SHtml {
     ajaxCall_*(jsCalcValue, jsContext, SFuncHolder(func))
 
   /**
+   * A wrapper around parseOpt for use in the jsonCall variants that will notify the developer on the console when
+   * parseOpt is unsuccessful at parsing the JSON.
+  **/
+  private def parseOptOrLog(rawJsonText: String) = {
```

Comment:
I think if this went into `lift-json` it would best be `parseOptOr(rawJsonText: String)(alternative: =>Option[JValue])` or something of the sort.